Description: Remove #!/bin/sh from attila.conf
 This patch removes the #!/bin/sh line from attila.conf since this file is not
 an executable script is sourced as a configuration file
Author: Roland Stigge <stigge@antcom.de>

--- alliance-5.1.1.orig/attila/etc/attila.conf
+++ alliance-5.1.1/attila/etc/attila.conf
@@ -1,6 +1,3 @@
-#!/bin/sh
-# -*- Mode: Shell-script -*-
-#
 # +------------------------------------------------------------------+
 # |        A l l i a n c e   C A D   S y s t e m                     |
 # |             T o o l   I n s t a l l e r                          |
